assets/datePicker.css

```diff
@@ -0,0 +1,117 @@
+/* purgecss start ignore */
+
+:root {
+  --tileSize: 47px;
```

Comment:
Why did u add `--` to the variables?

Reply:
This is the syntax to  declare a CSS variable

Reply:
CSS variables are not supported by all browsers, while the previous approach was a css-modules feature (which afaik got transpiled)

Reply:
Do we still need to support IE?

Reply:
nope, we do not support IE

Reply:
thanks, then we are good https://caniuse.com/css-variables

Reply:
Oh wasn't aware that a native support for this has been added...